src/pixel-ping.coffee

```diff
@@ -29,22 +29,36 @@ serialize = ->
 
 # Reset the `store`.
 reset = ->
+  old_store = store
```

Comment:
The style in the rest of this package (and Node/JS in general) seems to be `camelCase`.

Reply:
Good point.